spec/features/service_providers_spec.rb

```diff
@@ -23,6 +23,7 @@
 
       check 'email'
       check 'first_name'
+      check 'verified_at'
```

Comment:
@orenyk @zachmargolis updated these tests to make sure verified_at is available and selectable

Reply:
Great!